src/Controller.php

```diff
@@ -341,20 +341,20 @@ public function afterSave(Model $m, $is_update)
             $d = $this->getDiffs($m);
             foreach ($d as $f => list($f0, $f1)) {
                 if (
-                    isset($a['request_diff'][$f][1])
-                    && $a['request_diff'][$f][1] === $f1
+                    isset($a->get('request_diff')[$f][1])
+                    && $a->get('request_diff')[$f][1] === $f1
                 ) {
                     unset($d[$f]);
                 }
             }
-            $a['reactive_diff'] = $d;
+            $a->set('reactive_diff', $d);
 
-            if ($a['reactive_diff']) {
+            if ($d) {
```

Comment:
`count() > 0` would be more clear

Reply:
done but investigating further there was a logic error about comparison i extend. @mvorisek try to simplify it, i know you can :smile:

Reply:
wdym?